code/modules/client/client_procs.dm

```diff
@@ -536,6 +536,10 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	return ..()
 
 /client/Destroy()
+	gc_destroyed = world.time
+	if (!QDELING(src))
```

Comment:
Isn't this redundant? Only time it'd fail is if `world.time == 0` and it should be impossible for clients to connect then.

IIRC the implementation of `QDELING(X)` is `(##X.gc_destroyed)`

Reply:
This serves two purposes:

1. It acts as an explanation for the line above it
2. If `QDELING` definition changes (which it definitely reasonably could), we will know if clients no longer get marked as `QDELING`, something we can easily do in a test.